router/throttler/factory_test.go

```diff
@@ -71,6 +74,27 @@ func TestFactory(t *testing.T) {
 			return ta.getLimit() == int64(0)
 		}, 2*time.Second, 100*time.Millisecond)
 	})
+
+	t.Run("check stats when adaptive is enabled", func(t *testing.T) {
+		config := config.New()
+		config.Set("Router.throttler.adaptive.enabled", true)
+		maxLimit := int64(300)
+		window := 2 * time.Second
+		config.Set("Router.throttler.adaptive.maxLimit", maxLimit)
+		config.Set("Router.throttler.adaptive.minLimit", int64(100))
+		config.Set("Router.throttler.adaptive.timeWindow", window)
+		ctrl := gomock.NewController(t)
+		mockStats := mock_stats.NewMockStats(ctrl)
+		mockMeasurement := mock_stats.NewMockMeasurement(ctrl)
+		mockStats.EXPECT().NewTaggedStat("throttling_rate_limit", stats.GaugeType, gomock.Any()).Times(1).Return(mockMeasurement)
+		mockStats.EXPECT().NewTaggedStat("adaptive_throttler_limit_factor", stats.GaugeType, gomock.Any()).Times(1).Return(mockMeasurement)
+		mockMeasurement.EXPECT().Gauge(maxLimit / getWindowInSecs(window)).Times(1)
+		mockMeasurement.EXPECT().Gauge(float64(1)).AnyTimes()
+		f, err := NewFactory(config, mockStats)
+		require.NoError(t, err)
+		defer f.Shutdown()
+		f.Get("destName", "destID")
```

Comment:
We can explicitly declare that we ignore it?
```suggestion
		_ = f.Get("destName", "destID")
```

Reply:
Done